AI webinar initial draft (#2276)
FlowFuse/website: src/webinars/2024/bringing-ai-to-nodered.md

---
title: Bringing AI to Node-RED with FlowFuse Assistant
subtitle: Join us for this workshop on infusing Node-RED with AI within FlowFuse.
image:
date: 
time: 
duration: 60
hosts: ["nick-oleary"]
hubspot:
    formId:
---

**In this webinar, FlowFuse CTO and Node-RED creator Nick O'Leary will show how AI is being brought into the Node-RED world within FlowFuse.**

<!--more-->

There's no denying the place AI has in the software development world. Helping users be more productive and creating richer solutions using the power of AI.

> 

Reviewer: Add a short introduction of what FlowFuse Assistant is and what it enables, what you can achieve etc.

Author: Have expanded a bit more on the FlowFuse Assistant.

> In this webinar, Nick O'Leary will explore the different ways AI can improve your Node-RED applications - from help create the flows using the new FlowFuse Assistant, to building flows that leverage the AI to its full potential.

Reviewer: Mention the ideal target audience of this webinar..."This webinar is ideal for....who are looking to....

---
title: Bringing AI to Node-RED with FlowFuse Assistant

> subtitle: Join us for this workshop on infusing Node-RED with AI within FlowFuse.

Reviewer: ```suggestion
subtitle: Join us for this live webinar on infusing Node-RED with AI within FlowFuse.
```

image:

> date: 

Reviewer: ```suggestion
date: 2024-07-31
```

> time: 

Reviewer: ```suggestion
time: 17:00 CET (11:00am ET) 
```

duration: 60
hosts: ["nick-oleary"]
hubspot:

>     formId:

Reviewer: ```suggestion
    formId: febea57b-ef8d-437e-b1be-43f419bff291
```
